src/page/interactive_bom_button.jsx

```diff
@@ -0,0 +1,24 @@
+const React = require('react')
+const semantic = require('semantic-ui-react')
+
+const {folder} = require('../zip-info.json')
+const ibomUrl = `https://kitspace.org/${folder}/interactive_bom.html`
+
+function InteractiveBOMButton() {
+  return (
+    <div className="InteractiveBOMButton">
+      <semantic.Menu compact borderless>
+        <div className="InteractiveBOMButton__container">
+          <div className="InteractiveBOMButton__link">
+            <semantic.Menu.Item as="a" href="interactive_bom.html">
+              <semantic.Icon size="big" name="microchip" />
```

Comment:
I think the icon looks cute but I don't think it's actually a good one for this button. Chips are everywhere on Kitspace so it doesn't really tell the user anything.